### PR TITLE
Fixes #926

### LINKED ISSUE #926: Over-head text doesn't obey whisper mechanics
#### Description of issue

See title. When people whisper, you can see everything they say above their head, unlike in the chatlog

#### Difference between expected and actual behavior

Overhead text is censored like the chatlog

#### Steps to reproduce

Be far away from someone whispering

#### Specific information for locating

Whispers

#### Issue bingo

- [x] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/NebulaSS13/Nebula/issues)

### PR DESCRIPTION
**Pull request by comma**
:cl:
tweak: Ha ha ha I am abusing my GitHub maint powers to make chinsky commit a changelog via the bot workflow. Science!
/:cl:

### CODE REVIEW

File: code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -295,6 +295,7 @@ proc/get_radio_key_from_channel(var/channel)
 			if(O) //It's possible that it could be deleted in the meantime.
 				O.hear_talk(src, message, verb, speaking)
 
+	var/list/eavesdropers = list()
```
Comment (MistakeNot4892) [inline code/modules/mob/living/say.dm]: `eavesdroppers`

File: code/modules/mob/living/say.dm [hunk 2]
```diff
@@ -306,15 +307,17 @@ proc/get_radio_key_from_channel(var/channel)
...
-					speech_bubble_recipients |= M.client
+					eavesdropers |= M.client
 
 		for(var/obj/O in eavesdroping)
 			spawn(0)
 				if(O) //It's possible that it could be deleted in the meantime.
 					O.hear_talk(src, stars(message), verb, speaking)
 
-	INVOKE_ASYNC(GLOBAL_PROC, .proc/animate_speech_bubble, speech_bubble, speech_bubble_recipients, 30)
+	INVOKE_ASYNC(GLOBAL_PROC, .proc/animate_speech_bubble, speech_bubble, speech_bubble_recipients + eavesdropers, 30)
```
Comment (MistakeNot4892) [inline code/modules/mob/living/say.dm]: Would `|` be more appropriate?

File: code/modules/overmap/ships/computers/helm.dm [hunk 3]
```diff
@@ -201,6 +201,9 @@ LEGACY_RECORD_STRUCTURE(all_waypoints, waypoint)
 		autopilot = !autopilot
 
 	if (href_list["manual"])
+		if(!isliving(user))
+			to_chat(user, SPAN_WARN("You can't use this."))
```
Comment (MistakeNot4892) [inline code/modules/overmap/ships/computers/helm.dm]: `SPAN_WARNING`

File: code/modules/mob/living/say.dm (hunk 2 above)
Comment (comma) [reply code/modules/mob/living/say.dm]: Yeah changed to that

File: code/modules/mob/living/say.dm (hunk 1 above)
Comment (comma) [reply code/modules/mob/living/say.dm]: Fixed

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**